lib/addons/cloudwatch-adot-addon/collector-config-cloudwatch.ytpl

```diff
@@ -338,13 +338,7 @@ spec:
           enabled: true
         dimension_rollup_option: NoDimensionRollup
         parse_json_encoded_attr_values: [Sources, kubernetes]
-        metric_declarations:
```

Comment:
Agree that hard-coding to just some dimensions does not make sense. However, what will be the path now to limit it? I see this applicable to every single production use case. 

I suggest make it as a configurable set of dimensions to the users which are part of the properties.

Reply:
Fixed for custom metrics.